libs/components/lookup/src/lib/modules/selection-modal/selection-modal.component.html

```diff
@@ -29,6 +29,10 @@
           <sky-toolbar-view-actions>
             <button
               *ngIf="context.showAddButton"
+              [attr.aria-label]="
+                'skyux_lookup_show_more_add_button_aria_label'
+                  | skyLibResources : context.selectionDescriptor
+              "
```

Comment:
Should this text also be tested in the existing unit tests? looks good otherwise.

Reply:
added